resources/measures/upgrade_hvac_chiller/measure.rb

```diff
@@ -0,0 +1,1032 @@
+    when 'Constant Flow'
+      coeff_a = 0.0
+      coeff_b = 1.0
+      coeff_c = 0.0
+      coeff_d = 0.0
+    when 'Riding Curve'
+      coeff_a = 0.0
+      coeff_b = 3.2485
+      coeff_c = -4.7443
+      coeff_d = 2.5294
+    when 'VSD No Reset'
+      coeff_a = 0.0
+      coeff_b = 0.5726
+      coeff_c = -0.301
+      coeff_d = 0.7347
+    when 'VSD DP Reset'
+      coeff_a = 0.0
+      coeff_b = 0.0205
+      coeff_c = 0.4101
+      coeff_d = 0.5753
+    else
+      runner.registerError("Pump control type '#{control_type}' not recognized, pump coefficients will not be changed.")
+      return false
+    end
+
+    if debug_verbose
+      runner.registerInfo("### coeff_a = #{coeff_a}")
+      runner.registerInfo("### coeff_b = #{coeff_b}")
+      runner.registerInfo("### coeff_c = #{coeff_c}")
+      runner.registerInfo("### coeff_d = #{coeff_d}")
+      runner.registerInfo('### ------------------------------------------------------')
+    end
+
+    # Set the coefficients
+    pump_variable_speed.setCoefficient1ofthePartLoadPerformanceCurve(coeff_a)
+    pump_variable_speed.setCoefficient2ofthePartLoadPerformanceCurve(coeff_b)
+    pump_variable_speed.setCoefficient3ofthePartLoadPerformanceCurve(coeff_c)
+    pump_variable_speed.setCoefficient4ofthePartLoadPerformanceCurve(coeff_d)
+    pump_variable_speed.setPumpControlType('Intermittent')
+
+    # Append the control type to the pump name
+    # self.setName("#{self.name} #{control_type}")
+
+    return true
+  end
+
+  # Applies the condenser water temperatures to the plant loop based on Appendix G.
+  #
+  # hard-coding this because of https://github.com/NREL/openstudio-standards/issues/1915
+  # @param plant_loop [OpenStudio::Model::PlantLoop] plant loop
+  # @return [Boolean] returns true if successful, false if not
+  def plant_loop_apply_prm_baseline_condenser_water_temperatures(plant_loop)
+    sizing_plant = plant_loop.sizingPlant
+    loop_type = sizing_plant.loopType
+    return true unless loop_type == 'Condenser'
+
+    # Much of the thought in this section came from @jmarrec
+
+    # Determine the design OATwb from the design days.
+    # Per https://unmethours.com/question/16698/which-cooling-design-day-is-most-common-for-sizing-rooftop-units/
+    # the WB=>MDB day is used to size cooling towers.
+    summer_oat_wbs_f = []
+    plant_loop.model.getDesignDays.sort.each do |dd|
+      next unless dd.dayType == 'SummerDesignDay'
+      next unless dd.name.get.to_s.include?('WB=>MDB')
+
+      if plant_loop.model.version < OpenStudio::VersionString.new('3.3.0')
+        if dd.humidityIndicatingType == 'Wetbulb'
+          summer_oat_wb_c = dd.humidityIndicatingConditionsAtMaximumDryBulb
+          summer_oat_wbs_f << OpenStudio.convert(summer_oat_wb_c, 'C', 'F').get
+        else
+          OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.PlantLoop', "For #{dd.name}, humidity is specified as #{dd.humidityIndicatingType}; cannot determine Twb.")
```

Comment:
I'm not crazy about leaving these OpenStudio log messages as-is from the original standards code. I think they would make troubleshooting this measure difficult. I suggest changing them to `runner` warning/info/error

Reply:
ah. totally. all replaced!